Softeq.XToolkit.WhiteLabel.Droid/Dialogs/DroidFragmentDialogService.cs

```diff
@@ -73,7 +73,7 @@ public class DroidFragmentDialogService : IDialogsService
 
             var result = resultObject is TResult convertedResult
                 ? convertedResult
-                : default;
+                : default!;
```

Comment:
This is the first time I see something like that, can you please explain?

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-forgiving